.github/ISSUE_TEMPLATE.md

```diff
@@ -1,15 +1,28 @@
-<!-- BUGS: Please use this template. -->
+<!-- BUGGY OR UGLY? Please use this template.
 
-**Prettier Version:** 1.6.1 / master
+Tip! Don't write this stuff manually.
 
-**Code**
+1. Go to https://prettier.io/playground
+2. Paste your code and set options
+3. Press the "Report issue" button in the lower right
 
-([playground](https://prettier.io/playground/#.....))
+-->
 
-```js
+**Prettier 1.6.1**
+[Playground link](https://prettier.io/playground/#.....)
+```
+# Options (if any):
+--single-quote
+```
+
+**Input:**
+```jsx
 // code snippet
 ```
 
-**Expected behavior:**
+**Output:**
+```jsx
```

Comment:
Can we have a blank line after the `**Title:**`? as it looks more comfortable. 

![image](https://user-images.githubusercontent.com/8341033/30406217-10130e1a-9923-11e7-88c3-31dcf5d06b5d.png)

Reply:
The markdown does look a bit tider, but:

- I don't like the extra spacing between "Prettier 1.6.1" and "Playground link" in the rendered HTML.
- It makes the issue template appear longer.

Reply:
Oh! I forgot it'll cause additional blank line in HTML.